pkg/helm/helm.go

```diff
@@ -156,7 +154,11 @@ func NewHelmerImpl(chartsPath string) *Impl {
 func (r *Impl) Uninstall(cfg *action.Configuration, releaseName string) (*release.UninstallReleaseResponse, error) {
 	uninstall := action.NewUninstall(cfg)
 	uninstall.Wait = true
-	uninstall.Timeout = TIMEOUT
+	interval, err := environment.GetHelmWaitTimeout()
+	if err != nil {
```

Comment:
The idea is to use the default, not return the error.

Reply:
Fixed. Thanks